src/main/java/games/strategy/triplea/util/AiBonusIncomeUtils.java

```diff
@@ -0,0 +1,38 @@
+package games.strategy.triplea.util;
```

Comment:
Got it; makes sense.  There's currently three utility classes in `g.s.triplea.delegate`.  If `AiBonusIncomeUtils` is a similar type of utility, maybe extracting them all to a `g.s.triplea.delegate.util` (or something) package would be appropriate in the future?